evervault-inputs/src/main/java/com/evervault/sdk/input/ui/PaymentCardInputScope.kt

```diff
@@ -0,0 +1,82 @@
+package com.evervault.sdk.input.ui
+
+import androidx.compose.material3.TextFieldColors
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.TextStyle
+
+interface PaymentCardInputScope {
```

Comment:
Moved out from `PaymentCardInput` and added new composable methods commented below